packages/sdk/tasks/deposit-erc20.ts

```diff
@@ -127,7 +141,14 @@ task('deposit-erc20', 'Deposits WETH9 onto L2.')
     )
 
     const l2ChainId = await l2Signer.getChainId()
-    const contractAddrs = CONTRACT_ADDRESSES[l2ChainId]
+    let contractAddrs = CONTRACT_ADDRESSES[l2ChainId]
+    if (args.l1ContractsJsonPath) {
+      const data = await fs.readFile(args.l1ContractsJsonPath)
```

Comment:
You can `require` a json file right into an object

Reply:
Hmm, require doesn't seem to work in this case - reverting to use `fs` like before.